src/table/boot.rs

```diff
@@ -552,36 +567,6 @@ impl BootServices {
         unsafe { (self.set_watchdog_timer)(timeout, watchdog_code, data_len, data) }.into()
     }
 
-    /// Get the list of protocol interface [`Guids`][Guid] that are installed
-    /// on a [`Handle`].
-    pub fn protocols_per_handle(&self, handle: Handle) -> Result<ProtocolsPerHandle> {
-        let mut protocols = ptr::null_mut();
-        let mut count = 0;
-
-        let mut status = unsafe { (self.protocols_per_handle)(handle, &mut protocols, &mut count) };
-
-        if !status.is_error() {
-            // Ensure that protocols isn't null, and that none of the GUIDs
-            // returned are null.
-            if protocols.is_null() {
-                status = Status::OUT_OF_RESOURCES;
-            } else {
-                let protocols: &[*const Guid] = unsafe { slice::from_raw_parts(protocols, count) };
-                if protocols.iter().any(|ptr| ptr.is_null()) {
-                    status = Status::OUT_OF_RESOURCES;
-                }
-            }
-        }
-
-        status.into_with_val(|| {
-            let protocols = unsafe { slice::from_raw_parts_mut(protocols as *mut &Guid, count) };
-            ProtocolsPerHandle {
-                boot_services: self,
-                protocols,
-            }
-        })
-    }
-
```

Comment:
It looks like somehow it tries to remove these changes which already exist on `master` :thinking: 

Could you try [rebasing your branch](https://docs.github.com/en/get-started/using-git/about-git-rebase) then force pushing back to it? It should solve this issue.

Reply:
Alright